Sources/CassandraClient/Statement.swift

```diff
@@ -91,6 +91,21 @@ extension CassandraClient {
             try checkResult { cass_statement_set_paging_size(self.rawPointer, pagingSize) }
         }
 
+        /// Sets the starting page of the returned paginated results.
+        ///
+        /// The paging state token can be obtained by the `pagingStateToken()`
+        /// function on `Rows`.
+        ///
+        /// - Warning: The paging state should not be exposed to or come from
+        /// untrusted environments. The paging state could be spoofed and
+        /// potentially used to gain access to other data.
+        func setPagingStateToken(_ pagingStateToken: PagingStateToken) throws {
```

Comment:
Was this function supposed to be public?

Reply:
Yes, thanks!

Reply:
Resolved.